src/dataprotection/azext_dataprotection/manual/aaz_operations/backup_instance.py

```diff
@@ -0,0 +1,254 @@
+            schema_builder=self.BackupInstancesCreateOrUpdate._build_schema_on_200_201
+        )
+
+    class BackupInstancesValidateForBackup(_ValidateForBackup.BackupInstancesValidateForBackup):
+
+        @property
+        def content(self):
+            _content_value, _builder = self.new_content_builder(
+                self.ctx.args,
+                typ=AAZFreeFormDictArg,
+                value=self.ctx.vars.instance.properties,
+                typ_kwargs={"flags": {"required": True, "client_flatten": True}}
+            )
+            return {
+                "backupInstance": self.serialize_content(_content_value)
+            }
+
+    class BackupInstancesCreateOrUpdate(_Create.BackupInstancesCreateOrUpdate):
+
+        @property
+        def content(self):
+            _content_value, _builder = self.new_content_builder(
+                self.ctx.args,
+                value=self.ctx.vars.instance,
+            )
+            return self.serialize_content(_content_value)
+
+
+class ValidateForRestore(_ValidateForRestore):
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        if cls._args_schema is not None:
+            return cls._args_schema
+        cls._args_schema = super(_ValidateForRestore, cls)._build_arguments_schema(cls, *args, **kwargs)     # pylint: disable=bad-super-call
+
+        _args_schema = cls._args_schema
+        _args_schema.backup_instance_name = AAZStrArg(
+            options=["-n", "--name", "--backup-instance-name"],
+            help="The name of the backup instance.",
+            required=True,
+            id_part="child_name_1",
+        )
+        _args_schema.resource_group = AAZResourceGroupNameArg(
+            help="Name of resource group. You can configure the default group using `az configure --defaults group=<name>`.",
+            required=True,
+        )
+        _args_schema.vault_name = AAZStrArg(
+            options=["--vault-name"],
+            help="The name of the backup vault.",
+            required=True,
+            id_part="name",
+        )
+        _args_schema.restore_request_object = AAZFreeFormDictArg(
+            options=["--restore-request-object"],
+            help="Gets or sets the restore request object.",
+            required=True,
+        )
+        return cls._args_schema
+
+    class BackupInstancesValidateForRestore(_ValidateForRestore.BackupInstancesValidateForRestore):
+
+        def __call__(self, *args, **kwargs):
+            request = self.make_request()
+            session = self.client.send_request(request=request, stream=False, **kwargs)
+            if session.http_response.status_code in [202]:
+                return self.client.build_lro_polling(
+                    self.ctx.args.no_wait,
+                    session,
+                    self.on_200,
+                    self.on_error,
+                    lro_options={"final-state-via": "location"},
```

Comment:
Please fix the swagger directly

Reply:
Planned the fix, but it will take some time as discussed offline. A similar workaround is already committed to the repo: https://github.com/Azure/azure-cli-extensions/blob/0b2cbaab7ecd11a8cd99eb5d455045abb22ee9c9/src/dataprotection/azext_dataprotection/manual/custom.py#L180.